src/openeo_grass_gis_driver/actinia_processing/config.py

```diff
@@ -25,3 +31,51 @@ class Config:
     # The database file that stores the actinia jobs
     ACTINIA_JOB_DB = "%s/.actinia_job_db_file.sqlite" % os.environ["HOME"]
     SECRET_KEY = "jaNguzeef4seiv5shahchimoo8teiLah"
+
+
+class Configfile:
+
+    def __init__(self):
+        """
+        This class will overwrite the config classes above when config files
+        named DEFAULT_CONFIG_PATH/**/*.ini exist.
+        On first import of the module it is initialized.
+        """
+
+        config = configparser.ConfigParser()
+        print("Loading config files: " + str(CONFIG_FILES) + " ...")
+        config.read(CONFIG_FILES)
+
+        if len(config) <= 1:
+            print("Could not find any config file, using default values.")
+            return
+
+        with open(GENERATED_CONFIG, 'w') as configfile:
```

Comment:
Is it correct that GENERATED_CONFIG is written out but never read?

Reply:
Yes - it is for debug reasons to see which default values were overwritten and actually used. There may be multiple config files which will be merged. In the worst case a config value can appear in multiple files (it shouldn't) but then the merged config is really helpfull.